### PR TITLE
Lesson9

### PR DESCRIPTION
**Pull request by irakishmar**
Description of class Company and class Worker

### CODE REVIEW

File: lesson9/company.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    def count_of_employee(self, new_count):
+        self.__count_of_employee = new_count
+
+    @property
+    def budget(self):
+        return self.__budget
+
+    @budget.setter
+    def budget(self, new_budget):
+        self.__budget = new_budget
```
Comment (bohdandovbsyh) [inline lesson9/company.py]: looks like I can set a negative budget. Or string budget, or tuple...

File: lesson9/company.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    @property
+    def actual_info_year(self):
+        return self.__actual_info_year
+
+    @actual_info_year.setter
+    def actual_info_year(self, new_actual_info_year):
+        self.__actual_info_year = new_actual_info_year
```
Comment (bohdandovbsyh) [inline lesson9/company.py]: same here...please add different validations

File: lesson9/company.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                if new_data[1] == key:
+                    new_data[1] = month_dict[key]
+            new_data = ' '.join(new_data)
+            return new_data
+
+        formatted_date = get_month(founding_date)
+        return cls(name, count_of_employee, formatted_date, main_office, budget,
+                   actual_info_year)
+
+
```
Comment (bohdandovbsyh) [inline lesson9/company.py:99]: Please add some functions of a class.

File: lesson9/worker.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    def name(self, new_name):
+        self.__name = new_name
+
+    @property
+    def birth_year(self):
+        return self.__birth_year
+
+    @birth_year.setter
+    def birth_year(self, new_birth_year):
+        self.__birth_year = new_birth_year
```
Comment (bohdandovbsyh) [inline lesson9/worker.py]: please add validations

File: lesson9/worker.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+    @property
+    def salary(self):
+        return self.__salary
+
+    @salary.setter
+    def salary(self, new_salary):
+        self.__salary = new_salary
```
Comment (bohdandovbsyh) [inline lesson9/worker.py]: please add validations

File: lesson9/company.py (hunk 1 above)
Comment (irakishmar) [reply lesson9/company.py]: done

File: lesson9/company.py (hunk 2 above)
Comment (irakishmar) [reply lesson9/company.py]: done

File: lesson9/company.py (hunk 3 above)
Comment (irakishmar) [reply lesson9/company.py:99]: done

File: lesson9/worker.py (hunk 4 above)
Comment (irakishmar) [reply lesson9/worker.py]: done

File: lesson9/worker.py (hunk 5 above)
Comment (irakishmar) [reply lesson9/worker.py]: done

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**